tests/webpack.config.js

```diff
@@ -1,9 +1,9 @@
 var path = require('path');
-// `CheckerPlugin` is optional. Use it if you want async error reporting.
-// We need this plugin to detect a `--watch` mode. It may be removed later
-// after https://github.com/webpack/webpack/issues/3460 will be resolved.
-var CheckerPlugin = require('awesome-typescript-loader').CheckerPlugin;
+var ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
```

Comment:
I assume the fork version speeds things up a lot?

Reply:
from plain ts-loader, yes. it's basically doing some of the same things that awesome-ts-loader was doing to speed things up.